backend/danswer/connectors/file/utils.py

```diff
@@ -0,0 +1,58 @@
+import os
+import shutil
+import time
+import uuid
+from pathlib import Path
+from typing import Any
+from typing import IO
+
+_TEMP_LOCATION = Path("/tmp/file_connector")
+_FILE_AGE_CLEANUP_THRESHOLD = 3  # hours
```

Comment:
probably it's fine to not have any additional logic and if we cant index the whole zip in 3 hours its just not valid?

I'm not sure about that assumption actually...
Can we delete files as they get handled and not delete the ones that failed so we can go back and see why?

Reply:
Ok let's not increase complexity, let's just set the cleanup age to a week